packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -486,6 +480,12 @@ SwipeableDrawer.propTypes = {
    * @ignore
    */
   variant: PropTypes.oneOf(['permanent', 'persistent', 'temporary']),
+  /**
+   * Defines, from which (average) velocity on, the swipe is
+   * defined as complete although hysteresis isn't reached.
+   * Good threshold is between 0.5 - 2 px/ms
+   */
+  velocity: PropTypes.number,
```

Comment:
(we could also call the prop something linke "fling", like in the google comment [line 116](https://android.googlesource.com/platform/frameworks/support/+/407e01608ccafe5dc24f82608583f71c34312f9c/v4/java/android/support/v4/widget/DrawerLayout.java#116)

Reply:
top! 👍